lib/views/facebook_login_form.jade

```diff
@@ -16,6 +16,8 @@ script(type='text/javascript').
         var queryStr = window.location.search.replace('?', '');
         // TODO make dynamic
         if (queryStr) {
+          // Remove existing access token from query string.
```

Comment:
Same as my previous comment. Why do we want to remove this? Please add a comment describing why.